Additional reasons why Requests using Redirect URI Scheme cannot be signed (#456)
openid/OpenID4VP: openid-4-verifiable-presentations-1_0.md

This specification defines the following Client Identifier Schemes, followed by the examples where applicable: 

> * `redirect_uri`: This value indicates that the Client Identifier (without the prefix `redirect_uri:`) is the Verifier's Redirect URI (or Response URI when Response Mode `direct_post` is used). The Verifier MAY omit the `redirect_uri` Authorization Request parameter (or `response_uri` when Response Mode `direct_post` is used). All Verifier metadata parameters MUST be passed using the `client_metadata` parameter defined in (#new_parameters). Example Client Identifier value is `redirect_uri:https://client.example.org/cb`.

Author: ```suggestion
* `redirect_uri`: This value indicates that the Client Identifier (without the prefix `redirect_uri:`) is the Verifier's Redirect URI (or Response URI when Response Mode `direct_post` is used). The Verifier MAY omit the `redirect_uri` Authorization Request parameter (or `response_uri` when Response Mode `direct_post` is used). All Verifier metadata parameters MUST be passed using the `client_metadata` parameter defined in (#new_parameters). An example Client Identifier value is `redirect_uri:https://client.example.org/cb`. Requests using the `redirect_uri` Client Identifier cannot be signed because there is no method for the Wallet to obtain a trusted key for verification. This scheme lacks a mechanism to communicate the signing key, rendering it unsuitable for signed requests. Therefore, implementations requiring signed requests MUST NOT use the `redirect_uri` Client ID scheme. Due to the absence of a defined method for key management, the `redirect_uri` scheme is not intended for production use.
```

Reviewer: I think we can probably say the same thing with fewer words and fewer words is generally good :) 

```suggestion
* `redirect_uri`: This value indicates that the Client Identifier (without the prefix `redirect_uri:`) is the Verifier's Redirect URI (or Response URI when Response Mode `direct_post` is used). The Verifier MAY omit the `redirect_uri` Authorization Request parameter (or `response_uri` when Response Mode `direct_post` is used). All Verifier metadata parameters MUST be passed using the `client_metadata` parameter defined in (#new_parameters). An example Client Identifier value is `redirect_uri:https://client.example.org/cb`. Requests using the `redirect_uri` Client Identifier Scheme cannot be signed because there is no method for the Wallet to obtain a trusted key for verification. Therefore, implementations requiring signed requests cannot use the `redirect_uri` Client ID scheme.
```

> The following is a non-normative example of an unsigned request with this Client Identifier Scheme:

Author: ```suggestion
The following is a non-normative example of an unsigned request with the `redirect_uri` Client Identifier Scheme:
```

   [[ To be removed from the final specification ]]

   -25
   

>    * authz requests using redirect_uri scheme can be signed

Author: ```suggestion
   * additional reasons why requests using redirect_uri scheme cannot be signed
```

Reviewer: ```suggestion
   * clarify why requests using redirect_uri scheme cannot be signed
```
